src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -1835,11 +1835,15 @@ protected function prepareRequestUri()
             $this->server->remove('UNENCODED_URL');
             $this->server->remove('IIS_WasUrlRewritten');
         } elseif ($this->server->has('REQUEST_URI')) {
-            $requestUri = $this->server->get('REQUEST_URI');
             // HTTP proxy reqs setup request URI with scheme and host [and port] + the URL path, only use URL path
-            $schemeAndHttpHost = $this->getSchemeAndHttpHost();
-            if (0 === strpos($requestUri, $schemeAndHttpHost)) {
-                $requestUri = substr($requestUri, \strlen($schemeAndHttpHost));
+            $parsedUri = parse_url($this->server->get('REQUEST_URI'));
```

Comment:
Super minor comment: `$parsedUri` looks like a slightly changed URI to me ... but `parse_url()` returns the URI parts or components, not the URI itself. Maybe renaming this variable to `$uriParts` or `$uriComponents` would make it easier to understand? Or maybe not! Thanks!

Reply:
Thanks @javiereguiluz, nice idea, to be consistent with the `parse_url` PHP function documentation, I renamed `$parsedUri` to `$uriComponents` :+1: